ui/app/AppLayouts/Wallet/popups/swap/SwapSignModal.qml

```diff
@@ -45,8 +47,8 @@ SignTransactionModalBase {
 
     title: qsTr("Sign Swap")
     //: e.g. (swap) 100 DAI to 100 USDT
-    subtitle: qsTr("%1 %2 to %3 %4").arg(formatBigNumber(fromTokenAmount, 4)).arg(fromTokenSymbol) // FIXME get the correct number of decimals to display from the "symbol"
-        .arg(formatBigNumber(toTokenAmount, 4)).arg(toTokenSymbol)
+    subtitle: qsTr("%1 to %2").arg(root.currencyStore.formatCurrencyAmount(fromTokenAmount, fromTokenSymbol)) // FIXME get the correct number of decimals to display from the "symbol"
```

Comment:
the comment is no longer valid, right? we're using the correct format with this function